versionist.conf.js

```diff
@@ -0,0 +1,59 @@
+var execSync = require('child_process').execSync;
+
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
```

Comment:
Ah, so I guess you were probably copy'n'pasting round an old version of a variant of versionist's `versionist.conf.js`, and the 'upstream version' of `versionist.conf.js` has since been updated. And I guess this will continue to be a problem for any resin.io projects that are using versionist / VersionBot but aren't using NPM, so I wonder if https://github.com/resin-io/versionist/issues/84 needs to be expanded into a more flexible long-term solution...?